src/icons.scss

```diff
@@ -441,6 +447,11 @@
       @extend .fa-server;
       @extend .text-danger;
     }
+    &-disabled {
+      @extend .fa;
+      @extend .fa-server;
+      @extend .text-warning;
+    }
```

Comment:
One is a circle for the list of hosts, the other one is a picture of a server (displayed next to the host's name in the host details)